src/static/js/language.js

```diff
@@ -0,0 +1,42 @@
+const supportedLang = ['en', 'ja'];
+
+const langSwitcher = document.querySelector('.language-switcher');
+
+langSwitcher.addEventListener('change', (e) => {
+  if (e.target.value) {
+    window.location.href =
+      updatePath(window.location.pathname, e.target.value);
+  }
+});
+
+function updatePath(path, value) {
```

Comment:
Does this only work for the home page, or does it also support subdirectories? For example, this URL: `/2019/outline`.

Ideally we would use the Jinja [url_for](http://flask.pocoo.org/docs/1.0/api/#flask.url_for) method to construct the new URLs dynamically in the base template.